app/views/host_aggregate/add_host_select.html.haml

```diff
@@ -0,0 +1,37 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "hostAggregateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Add Host to Host Aggregate')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Host')
+      .col-md-8
+        = select_tag("host_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @host_choices.sort),
+                      "ng-model"                    => "hostAggregateModel.host_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
+          = button_tag(_("Add"),
+                       :class        => "btn btn-primary",
+                       "ng-click"    => "addHostClicked()",
+                       "ng-disabled" => "angularForm.$pristine || angularForm.$invalid",
+                       "ng-class"    => "{ disabled: angularForm.$pristine || angularForm.$invalid}")
+          = button_tag(_("Reset"),
+                       :class        => "btn btn-primary",
+                       "ng-click"    => "resetClicked()",
```

Comment:
That's the way reset was done for the features I modeled this on. I thought that was the point of reset.